site/_data/v1-1-0-toc.yml

```diff
@@ -25,6 +25,8 @@ toc:
         url: /gcp-config
       - page: Restic setup
         url: /restic
+      - page: Run locally
```

Comment:
The page should probably be called "run locally in development" since we tell users to build from the master branch

Reply:
How about I add that at the top of the page itself, just to avoid having this menu item be so long?

Reply:
I'd prefer to not include this in the `Install` section, given that it's mainly a development topic, and it's in the `Contribute` section already.

Reply:
This is the last thing for me, then LGTM!

Reply:
I thought I had removed it. Removing!